main.go

```diff
@@ -84,13 +86,16 @@ func connectOnce() {
 	irccon.VerboseCallbackHandler = false
 	irccon.Debug = false
 	irccon.UseTLS = true
-	irccon.AutoReconnect = false
+	host, _, err := net.SplitHostPort(servertls)
+	if err != nil {
+		log.Panicf("parse server: %s\n", err)
```

Comment:
I'd add "irc" in the error message, might be clearer when hitting errors

Reply:
Done.